dir.traversal.targets

```diff
@@ -54,56 +54,20 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
-  <!-- FilterProjectsToTest will filter the project list according to the FilterToTestTFM value -->
-  <Target Name="FilterProjectsToTest"
-          BeforeTargets="TestAllProjects"
-          Condition="$(MSBuildProjectName.EndsWith('tests'))">
-
-    <ItemGroup>
-      <Project>
-        <!-- default to DefaultTestTFM defined in root dir.props if TestTFMs aren't set on the project -->
-        <TestTFMs Condition="'%(Project.TestTFMs)'==''">$(DefaultTestTFM)</TestTFMs>
-      </Project>
-      <Project>
-        <!-- make sure every defined TestTFM value are surrounded by semicolons for easier search, e.g. ";net46;"" -->
-        <TestTFMs>;%(Project.TestTFMs);</TestTFMs>
-      </Project>
-
-      <!-- Include all projects which have TestTFM from the supported set -->
-      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.Identity)').Contains('.Tests'))">
```

Comment:
Yeah, I understand that, it still gets a false-positive for RemoteExecutorConsoleApp since that is src/Common/**tests**/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.csproj.